Node/puppeteer_vanilla/tests/example.spec.js

```diff
@@ -0,0 +1,32 @@
+const puppeteer = require('puppeteer');
+const { AxeDevToolsPuppeteer } = require('@axe-devtools/puppeteer');
+const Reporter = require('@axe-devtools/reporter').default;
+const reporter = new Reporter('Basic Test Suite', './a11y-results/json');
+var assert = require('assert');
+
+(async () => {
+    const browser = await puppeteer.launch({
+        headless: false,
+    });
+    const page = await browser.newPage();
+    await page.goto("https://broken-workshop.dequelabs.com/");
+    // Uncomment the below line if screenshot is required
+    // await page.screenshot({
+    //     path: 'example_'  + '.png'
+    // });
+    const results = await new AxeDevToolsPuppeteer(page).analyze();
+    console.log(results);
```

Comment:
why are we console logging results here?

Reply:
removed the console logging. Was just for the purpose of the user to see the results in the logs if needed by them.